### DISCUSSION TITLE
[Projects Beta] Is there any way to bulk add issues to a project?

### DISCUSSION THREAD

**Discussion by clapfootmark** [community/community#6597] [Projects and Issues]
So far it seems like the only way to add issues to a project (beta) is to either individual add them from the project screen (ctrl + space) or to assign the individually on an individual issue's screen. What I'd like to do is to be able to select a bunch of issues from the issues list view (or wherever) and dump them into a project.

Perhaps my use case isn't supported, but I have a 100 issues that I want to add, am I really expected to go through them 1 by 1 to add them to a project?

**Comment by imlazyeye** [comment]
+1 to this! Would really alleviate the burden of setting up a new project. Would love to see a widget that allowed me to add all issues / PRs that matched a given set of filters (and as bonus points, set the default fields for all of them).

**Comment by sol-mocha** [comment]
+1, the old github projects had the `Add Cards` button that let me drag in open issues to the projects. The lack of an equivalent has stopped me and my team from converting over...
